db/seeds.rb

```diff
@@ -29,3 +29,5 @@
                               food_truck: f)
   end
 end
+
+User.create(email: "admin@example.com", password: 'password', admin: true)
```

Comment:
```
user = User.find_or_initialize_by(email: "admin@example.com")
user.update_attributes(password: "password", admin: true)
```

Since your app is open source, anyone could see these credentials and log in as an administrator. How might you combat this?